### DISCUSSION TITLE
How do you configure it to make it join other gaws as well?

### DISCUSSION THREAD

**Discussion by Zorral** [Vedza/NitroSniperGo#234] [General]
Sorry for all the questions again, and I think I have managed to fix every one of them. Here is the bot that I found that works best: GiveawayBot#2381, while this one doesn't work at all Noumenon #7110. This code is really quite fast but when adding whitelisted words, it's not very consistent, and doesn't seem to work with anything else than a nitro gaw. Any fixes?

**Comment by Vedza** [comment]
That's the all point of the whitelist, only join giveaways with these words in it. So if you put Nitro in it, it will only join Nitro giveaway. Leave whitelist empty ([ ]) to disable that.

**Reply by Zorral** [reply]
I understand, but it doesn't seem to work when I add other words in it.

**Reply by Zorral** [reply]
wdym by disable chat?
